lib/ansible/plugins/doc_fragments/constructed.py

```diff
@@ -79,4 +79,8 @@ class ModuleDocFragment(object):
     type: boolean
     default: True
     version_added: '2.11'
+  notes:
+    - Inventories are not finalized at this stage, so the auto populated 'all' and 'ungrouped' groups will
+      only reflect what previous inventory sources explicitly added to them.
+    - Runtime 'magic variables' are not available during inventory construction. For example, 'groups' and 'hostvars' do not exist yet.
```

Comment:
```suggestion
notes:
  - Inventories are not finalized at this stage, so the auto populated 'all' and 'ungrouped' groups will
    only reflect what previous inventory sources explicitly added to them.
  - Runtime 'magic variables' are not available during inventory construction. For example, 'groups' and 'hostvars' do not exist yet.
```
I would probably use `C(...)` instead of `'...'` for `all`, `ungrouped`, `groups`, and `hostvars`.